src/battle_util.c

```diff
@@ -8673,6 +8668,12 @@ static inline u32 CalcMoveBasePower(u32 move, u32 battlerAtk, u32 battlerDef, u3
         if (RandomPercentage(RNG_FICKLE_BEAM, 30))
             basePower *= 2;
         break;
+    case EFFECT_LAST_RESPECTS:
+        if (GetBattlerSide(battlerAtk) == B_SIDE_PLAYER)
+            basePower += (basePower * min(100, gBattleResults.playerFaintCounter));
+        else
+            basePower += (basePower * min(100, gBattleResults.opponentFaintCounter));
+        break;
```

Comment:
To remove duplicate code, you can do something like:
```
u8 faintCounter = min(100, (GetBattlerSide(battlerAtk) == B_SIDE_PLAYER) ? gBattleResults.playerFaintCounter : gBattleResults.opponentFaintCounter)
basePower += basePower * faintCounter
```
```

Reply:
Or maybe an inline function to get a side's faint counter 🤔